modules/core/04-channel/v2/keeper/packet_test.go

```diff
@@ -227,25 +191,33 @@ func (suite *KeeperTestSuite) TestRecvPacket() {
 			path = ibctesting.NewPath(suite.chainA, suite.chainB)
 			path.SetupV2()
 
+			packetData := mockv2.NewMockPacketData(mockv2.ModuleNameA, mockv2.ModuleNameB)
+
+			timeoutTimestamp := uint64(suite.chainB.GetContext().BlockTime().Add(time.Hour).Unix())
+
+			// create standard packet that can be malleated
+			packet = types.NewPacket(0, path.EndpointA.ChannelID, path.EndpointB.ChannelID,
+				timeoutTimestamp, packetData)
+
 			// send packet
-			sequence, err := path.EndpointA.SendPacketV2(defaultTimeoutHeight, disabledTimeoutTimestamp, mock.Version, ibctesting.MockPacketData)
+			sequence, err := path.EndpointA.SendPacketV2(packet.TimeoutTimestamp, packet.Data...)
 			suite.Require().NoError(err)
 
-			packet = channeltypes.NewPacketWithVersion(ibctesting.MockPacketData, sequence, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, defaultTimeoutHeight, disabledTimeoutTimestamp, mock.Version)
+			packet.Sequence = sequence
```

Comment:
[nit] this seems a bit odd creating a packet, then modifying it after, why not just

```golang
sequence, err := path.EndpointA.SendPacketV2(timeoutTimestamp, packetData...)
suite.Require().NoError(err)
packet = types.NewPacket(sequnce, path.EndpointA.ChannelID, path.EndpointB.ChannelID, timeoutTimestamp, packetData)
```

Reply:
yea, got lazy fixing builds. Cleaned it up now